[2024-09-04 23:39:20,029:INFO:main:Welcome to my custom log]
[2024-09-06 11:59:11,632:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-06 11:59:11,636:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-06 11:59:11,637:INFO:common:created directory at: artifacts]
[2024-09-06 11:59:11,638:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-06 12:04:59,947:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-06 12:04:59,948:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-06 12:04:59,949:INFO:common:created directory at: artifacts]
[2024-09-06 12:04:59,950:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-06 12:08:54,851:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-06 12:08:54,852:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-06 12:08:54,853:INFO:common:created directory at: artifacts]
[2024-09-06 12:08:54,854:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-06 12:09:12,170:INFO:3798909177:artifacts/data_ingestion/data.zip download! with following info: 
 Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: CB6C:192D66:124719:169EDE:66DAA0EE
Accept-Ranges: bytes
Date: Fri, 06 Sep 2024 06:38:54 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10243-MAA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1725604735.817714,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: a3ca1f908d116fe771e01cf2f10c7c406c81843e
Expires: Fri, 06 Sep 2024 06:43:54 GMT
Source-Age: 234

]
[2024-09-07 14:03:11,596:INFO:main:>>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-07 14:03:11,596:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:03:11,596:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:03:11,596:INFO:common:created directory at: artifacts]
[2024-09-07 14:03:11,596:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-07 14:03:36,063:INFO:data_ingestion:artifacts/data_ingestion/data.zip download! with following info: 
 Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: B072:1B510:88193:BA739:66DC0FC8
Accept-Ranges: bytes
Date: Sat, 07 Sep 2024 08:33:14 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10250-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1725697993.306833,VS0,VE1055
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 3a89c14892c238afe494af1475bd1c7527d25d36
Expires: Sat, 07 Sep 2024 08:38:14 GMT
Source-Age: 1

]
[2024-09-07 14:03:36,302:INFO:main:>>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-07 14:49:58,456:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:49:58,461:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:49:58,461:INFO:common:created directory at: artifacts]
[2024-09-07 14:49:58,461:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 14:51:59,823:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 14:54:28,842:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:54:28,842:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:54:28,847:INFO:common:created directory at: artifacts]
[2024-09-07 14:54:28,847:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 14:54:29,110:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 14:54:52,953:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:54:52,953:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:54:52,953:INFO:common:created directory at: artifacts]
[2024-09-07 14:54:52,953:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 14:54:53,221:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 14:57:01,243:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:57:01,243:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:57:01,243:INFO:common:created directory at: artifacts]
[2024-09-07 14:57:01,259:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 14:57:01,527:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 14:57:30,715:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:57:30,718:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:57:30,718:INFO:common:created directory at: artifacts]
[2024-09-07 14:57:30,718:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 14:57:31,021:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 14:58:23,617:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 14:58:23,617:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 14:58:23,617:INFO:common:created directory at: artifacts]
[2024-09-07 14:58:23,617:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 14:58:23,900:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 14:58:24,033:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 18:41:29,370:INFO:main:>>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-07 18:41:29,374:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 18:41:29,376:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 18:41:29,378:INFO:common:created directory at: artifacts]
[2024-09-07 18:41:29,378:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-07 18:41:50,691:INFO:data_ingestion:artifacts/data_ingestion/data.zip download! with following info: 
 Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 1848:18CEB:BA65F:F868F:66DC5102
Accept-Ranges: bytes
Date: Sat, 07 Sep 2024 13:11:31 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10235-MAA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1725714690.316302,VS0,VE1235
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 590b32e7b1563fdc9557ecf4d66f9efacbebbb18
Expires: Sat, 07 Sep 2024 13:16:31 GMT
Source-Age: 1

]
[2024-09-07 18:41:50,960:INFO:main:>>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-07 18:41:50,960:INFO:main:***********************]
[2024-09-07 18:41:50,960:INFO:main:>>>>>>> stage Prepare Base Model started <<<<<<<]
[2024-09-07 18:41:50,960:ERROR:main:PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "D:\Project Codes\Chicken_Disease_Classification\main.py", line 27, in <module>
    obj.main()
TypeError: PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given
[2024-09-07 18:44:56,624:INFO:main:>>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-07 18:44:56,624:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 18:44:56,640:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 18:44:56,640:INFO:common:created directory at: artifacts]
[2024-09-07 18:44:56,640:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-07 18:44:56,640:INFO:data_ingestion:File already exists of size: ~ 11345 KB]
[2024-09-07 18:44:56,809:INFO:main:>>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-07 18:44:56,809:INFO:main:***********************]
[2024-09-07 18:44:56,809:INFO:main:>>>>>>> stage Prepare Base Model started <<<<<<<]
[2024-09-07 18:44:56,809:ERROR:main:PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "D:\Project Codes\Chicken_Disease_Classification\main.py", line 27, in <module>
    obj.main()
TypeError: PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given
[2024-09-07 18:45:43,027:INFO:main:>>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-07 18:45:43,027:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 18:45:43,027:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 18:45:43,043:INFO:common:created directory at: artifacts]
[2024-09-07 18:45:43,043:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-07 18:45:43,043:INFO:data_ingestion:File already exists of size: ~ 11345 KB]
[2024-09-07 18:45:43,219:INFO:main:>>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-07 18:45:43,219:INFO:main:***********************]
[2024-09-07 18:45:43,219:INFO:main:>>>>>>> stage Prepare Base Model started <<<<<<<]
[2024-09-07 18:45:43,219:ERROR:main:PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "D:\Project Codes\Chicken_Disease_Classification\main.py", line 27, in <module>
    obj.main()
TypeError: PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given
[2024-09-07 18:46:49,073:INFO:stage_02_prepare_base_model:***********************]
[2024-09-07 18:46:49,073:INFO:stage_02_prepare_base_model:>>>>>>> stage Prepare Base Model started <<<<<<<]
[2024-09-07 18:46:49,089:ERROR:stage_02_prepare_base_model:PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "d:\Project Codes\Chicken_Disease_Classification\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 25, in <module>
    obj.main()
TypeError: PrepareBaseModelTrainingPipeline.main() takes 0 positional arguments but 1 was given
[2024-09-07 18:48:07,827:INFO:stage_02_prepare_base_model:***********************]
[2024-09-07 18:48:07,827:INFO:stage_02_prepare_base_model:>>>>>>> stage Prepare Base Model started <<<<<<<]
[2024-09-07 18:48:07,827:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 18:48:07,842:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 18:48:07,843:INFO:common:created directory at: artifacts]
[2024-09-07 18:48:07,845:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 18:48:08,209:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 18:48:08,359:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 18:48:08,484:INFO:stage_02_prepare_base_model:>>>>>>> stage Prepare Base Model completed <<<<<<<

x==========x]
[2024-09-07 18:48:49,502:INFO:main:>>>>>>> stage Data Ingestion stage started <<<<<<<]
[2024-09-07 18:48:49,510:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 18:48:49,510:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 18:48:49,510:INFO:common:created directory at: artifacts]
[2024-09-07 18:48:49,510:INFO:common:created directory at: artifacts/data_ingestion]
[2024-09-07 18:49:06,581:INFO:data_ingestion:artifacts/data_ingestion/data.zip download! with following info: 
 Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 1848:18CEB:BA65F:F868F:66DC5102
Accept-Ranges: bytes
Date: Sat, 07 Sep 2024 13:18:50 GMT
Via: 1.1 varnish
X-Served-By: cache-maa10231-MAA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1725715131.574660,VS0,VE249
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 78eaf00d01ea6f516f122ab097f72d12f4434737
Expires: Sat, 07 Sep 2024 13:23:50 GMT
Source-Age: 0

]
[2024-09-07 18:49:06,787:INFO:main:>>>>>>> stage Data Ingestion stage completed <<<<<<<

x==========x]
[2024-09-07 18:49:06,787:INFO:main:***********************]
[2024-09-07 18:49:06,787:INFO:main:>>>>>>> stage Prepare Base Model started <<<<<<<]
[2024-09-07 18:49:06,803:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 18:49:06,803:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 18:49:06,803:INFO:common:created directory at: artifacts]
[2024-09-07 18:49:06,803:INFO:common:created directory at: artifacts/prepare_base_model]
[2024-09-07 18:49:07,121:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 18:49:07,251:WARNING:saving_api:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2024-09-07 18:49:07,326:INFO:main:>>>>>>> stage Prepare Base Model completed <<<<<<<

x==========x]
[2024-09-07 19:06:58,631:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:06:58,636:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:06:58,636:INFO:common:created directory at: artifacts]
[2024-09-07 19:08:11,865:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:08:11,865:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:08:11,869:INFO:common:created directory at: artifacts]
[2024-09-07 19:09:13,104:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:09:13,104:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:09:13,104:INFO:common:created directory at: artifacts]
[2024-09-07 19:09:13,112:INFO:common:created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-09-07 19:09:13,112:INFO:common:created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-09-07 19:10:06,216:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:10:06,216:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:10:06,216:INFO:common:created directory at: artifacts]
[2024-09-07 19:10:06,223:INFO:common:created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-09-07 19:10:06,224:INFO:common:created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-09-07 19:10:17,680:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:10:17,680:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:10:17,680:INFO:common:created directory at: artifacts]
[2024-09-07 19:10:17,685:INFO:common:created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-09-07 19:10:17,687:INFO:common:created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-09-07 19:10:40,901:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:10:40,909:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:10:40,909:INFO:common:created directory at: artifacts]
[2024-09-07 19:10:40,909:INFO:common:created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-09-07 19:10:40,909:INFO:common:created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-09-07 19:11:34,716:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:11:34,716:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:11:34,716:INFO:common:created directory at: artifacts]
[2024-09-07 19:11:34,725:INFO:common:created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-09-07 19:11:34,725:INFO:common:created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2024-09-07 19:12:16,049:INFO:common:yaml file: config\config.yaml loaded successfully]
[2024-09-07 19:12:16,049:INFO:common:yaml file: params.yaml loaded successfully]
[2024-09-07 19:12:16,049:INFO:common:created directory at: artifacts]
[2024-09-07 19:12:16,057:INFO:common:created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2024-09-07 19:12:16,057:INFO:common:created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
